samples/boards/nrf52/mesh/onoff_level_app/src/main.c

```diff
@@ -0,0 +1,38 @@
+#include "common.h"
+
+void main(void)
+{
+	int err, rc = 0;
+
+	rc = nvs_init(&fs, FLASH_DEV_NAME, STORAGE_MAGIC);
+
+	if (rc) {
+		printk("Flash Init failed\n");
```

Comment:
missing return statement? If nvs_init failed I don't think it's save to do e.g. an nvs_read() call.